how-to/create-addon.rst

```diff
@@ -6,7 +6,7 @@ How to package a Django application as an addon
 Register the addon
 ------------------
 
-Before your addon can be uploaded, the Divio Cloud must be ready to receive it.
+Before your addon can be uploaded, Divio must be ready to receive it.
```

Comment:
_...the Divio Control Panel must be ready..._

Reply:
Done